include/external/block_timer.h

```diff
@@ -0,0 +1,30 @@
+//
+// Created by Alexander Taepper on 02.04.23.
+//
+
+#ifndef SILO_BLOCK_TIMER_H
+#define SILO_BLOCK_TIMER_H
+
+#include <chrono>
+
+template <typename Unit = std::chrono::microseconds, typename Clock = std::chrono::steady_clock>
+struct [[nodiscard]] BlockTimer {
+   using time_point_t = typename Clock::time_point;
+   using output_t = typename Clock::rep;
+
+   output_t& output;
+   time_point_t start;
+
+   explicit BlockTimer(output_t& ref)
```

Comment:
What is `ref`? From the name, I can not find out its meaning. Same with `output`.